LongWarOfTheChosen/Config/XComGameData_WeaponData.ini

```diff
@@ -1768,3 +1768,210 @@ SPARK_NOVA_SELF_BASEDAMAGE= (Damage=0, Spread = 0, PlusOne = 0, Crit = 0, Pierce
 FERALMEC_M1_PROXY_WPN_BASEDAMAGE=(Damage=2, Spread=0, PlusOne=25, Crit=1, Pierce=0, Shred=0, Tag="", DamageType="Projectile_Conventional")
 
 SECTOPOD_MARKOV_PROXY_WPN_BASEDAMAGE=(Damage=3, Spread = 0, PlusOne = 25, Crit = 1, Pierce = 0, Shred = 1, Tag = "", DamageType="Projectile_Conventional")
+
+[WotCWeaponsAndArmor.X2Item_Weapons]
+;Bukllpup stats
+BULLPUP_CONVENTIONAL_BASEDAMAGE=(Damage=3, Spread=0, PlusOne=50, Crit=1, Pierce=0, Shred=0, Tag="", DamageType="Projectile_Conventional")
+BULLPUP_MAGNETIC_BASEDAMAGE=(Damage=6, Spread=0, PlusOne=50, Crit=2, Pierce=0, Shred=0, Tag="", DamageType="Projectile_MagXCom")
+BULLPUP_BEAM_BASEDAMAGE=(Damage=8, Spread=0, PlusOne=50, Crit=4, Pierce=0, Shred=0, Tag="", Rupture = 1, DamageType="Projectile_BeamXCom")
+
+BULLPUP_CONVENTIONAL_AIM = 0
+BULLPUP_CONVENTIONAL_CRITCHANCE = 0
+BULLPUP_CONVENTIONAL_ICLIPSIZE = 3
+BULLPUP_CONVENTIONAL_ISOUNDRANGE = 30
```

Comment:
Just want to point out that this is slightly higher than SMG. I'm happy with that.

Reply:
Yeah I wanted them to have just a bit more ammo than smg since they can actually hold more.

Reply:
Sorry, was referring to sound range.

Reply:
Ahh, yeah that too. Forgot I had set that higher.